core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -706,22 +725,23 @@ private Task taskToBeRescheduled(Workflow workflow, Task task) {
 
     public Task getPendingTaskByWorkflow(String taskReferenceName, String workflowId) {
         return executionDAOFacade.getTasksForWorkflow(workflowId).stream()
-                .filter(NON_TERMINAL_TASK)
-                .filter(task -> task.getReferenceTaskName().equals(taskReferenceName))
-                .findFirst() // There can only be one task by a given reference name running at a time.
-                .orElse(null);
+            .filter(NON_TERMINAL_TASK)
+            .filter(task -> task.getReferenceTaskName().equals(taskReferenceName))
+            .findFirst() // There can only be one task by a given reference name running at a time.
+            .orElse(null);
     }
 
     private void endExecution(Workflow workflow) {
         Optional<Task> terminateTask = workflow.getTasks().stream()
-                .filter(t -> TERMINATE.name().equals(t.getTaskType()) && t.getStatus().isTerminal()
-                        && t.getStatus().isSuccessful())
-                .findFirst();
+            .filter(t -> TERMINATE.name().equals(t.getTaskType()) && t.getStatus().isTerminal()
+                && t.getStatus().isSuccessful())
+            .findFirst();
         if (terminateTask.isPresent()) {
             String terminationStatus =
-                    (String) terminateTask.get().getWorkflowTask().getInputParameters().get(Terminate.getTerminationStatusParameter());
+                (String) terminateTask.get().getWorkflowTask().getInputParameters()
```

Comment:
Is IntelliJ code format causing this?

Reply:
yes, i think we need to attach a style guide to the project so that this remains consistent.